.changelog/unreleased/improvements/ibc-relayer/4237-improve-tls-configuration.md

```diff
@@ -0,0 +1,4 @@
+- Only configure TLS if the scheme is HTTPS and if the endpoint is
+  IPv6, parse the endpoint to remove the brackets before setting it
+  as domain name.
```

Comment:
This should be under `bug-fixes`.

```suggestion
- Fix an issue where Hermes would fail to connect to gRPC servers over HTTP on an IPv6 address. HTTPS over IPv6 is still unsupported at the moment because of a bug in the underlying gRPC client library.
```

Reply:
Done in 9ddb477b8bc7fa330c99bf6677f3a730bf93d1aa